utils.go

```diff
@@ -30,20 +25,6 @@ func convert(i interface{}) interface{} {
 	return i
 }
 
-func getProjectGroupFromGitURL(gitURL string, projectName string) string {
-	pattern := regexp.MustCompile(`((git@[\w\.]+)):(?P<projectPath>[\w\.@\:/\-~]+)(\.git)(/)?`)
-	if !pattern.MatchString(gitURL) {
-		return ""
-	}
-
-	template := "$projectPath"
-	projectPath := pattern.ReplaceAllString(gitURL, template)
-	projectGroup := strings.TrimSuffix(strings.ToLower(projectPath), strings.ToLower(projectName))
-	projectGroup = strings.TrimSuffix(projectGroup, "/")
-
-	return projectGroup
-}
```

Comment:
*I'm kind of annoyed that this function existed to begin with. I can rest easier now that it's gone though.*